src/deviceContext.ts

```diff
@@ -312,29 +312,14 @@ export class DeviceContext implements IDeviceContext, vscode.Disposable {
             .then(async (fileUris) => {
                 if (fileUris.length === 0) {
                     let newSketchFileName = await vscode.window.showInputBox({
-                        value: "my-sketch.ino",

```

Comment:
Did we figure out a reason why this default name caused issues?

Reply:
Or why it was disallowed?

Reply:
It didn't pass the incorrect regex below.

Reply:
Understood. I guess I was more curious about if there was a reason why we are restricting a filename of that style?

Reply:
We're not anymore. Notice that the regex itself has also changed based on EW's TODO comment.